### PR TITLE
README: add contact us details

### PR DESCRIPTION
**Pull request by zimbatm**
This adds a section to the README that explains how to contact us, with attached expectations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,16 @@ The "Stiching NixOS Foundation" is a registered non-profit organisation at the C
...
+
+## Contact us
+
+In order to bring items to our attention, you have the following mechanisms
+available:
+
+* Talk to the individual members if you want to pick our brain. We try to make
+    ourselves available.
+* For private matters, please contact
+    [foundation@nixos.org](mailto:foundation@nixos.org).
```
[the remark this answers is not in the document]
Comment (domenkozar) [reply README.md:32]: I don't know where it's configured, but I do get the emails.

[the remark this answers is not in the document]
Comment (zimbatm) [reply README.md:32]: Maybe the @NixOS/infra team would know

File: README.md [hunk 2]
```diff
@@ -20,3 +20,16 @@ The "Stiching NixOS Foundation" is a registered non-profit organisation at the C
...
+* If you need something to be done, please open [an
+    issue](https://github.com/NixOS/foundation/issues/new). New issues will be
+    reviewed during our Biweely meetings.
```
[the remark this answers is not in the document]
Comment (vcunat) [reply README.md:35]: Nit: missing "k" in "Biweely".

[the remark this answers is not in the document]
Comment (domenkozar) [reply README.md:35]: Fixed, thanks!

**Pull request merged**